main.tf

```diff
@@ -369,29 +369,37 @@ module "private_ecr_github_actions_upload_credentials_neuroagent" {
 module "launch_system" {
   source = "./private-ecr-repo"
 
-  repository_name            = "launch-system"
-  allowed_to_pull_identities = ["arn:aws:iam::992382665735:role/launch20251017122840582600000005", "arn:aws:iam::671250183987:role/launch20251028124644152500000004"]
+  repository_name = "launch-system"
+  allowed_to_pull_identities = [
+    "arn:aws:iam::671250183987:role/launch20251028124644152500000004", # production
```

Comment:
launch_system does not need to run in staging? The way it is with this commit, only prod will be able to pull. If that's what you want all good for me :)